packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -1275,6 +1279,23 @@ function prepareFreshStack(root, expirationTime) {
   }
 }
 
+export function unstable_avoidThisRender(value: Thenable) {
```

Comment:
Drop this indirection. Just call `unstable_avoidThisRenderImpl` directly from ReactFiberHooks.

Reply:
I placed it here to have access to `workInProgressRoot`, `renderExpirationTime`, and `workInProgress`. Only `workInProgress` is in scope inside `ReactFiberHooks` (as `currentlyRenderingFiber`) Is there a simple way to get a hold of the former two?